lib/helpers/fields/layer-fields.js

```diff
@@ -0,0 +1,19 @@
+const Fields = require('./fields')
+
+class LayerFields extends Fields {
+  static create (inputOptions) {
```

Comment:
Default `{}`?

Reply:
not needed because of the normalize method on the parent class - already handle the default value.